.github/workflows/publish-docs.yml

```diff
@@ -0,0 +1,42 @@
+name: Publish mkdocs
+on:
+  pull_request:
+  push:
+    branches:
+      - master
+jobs:
+  deploy:
+    runs-on: ubuntu-20.04
+    steps:
+      - uses: actions/checkout@2541b1294d2704b0964813337f33b291d3f8596b #v3.0.2
+        with:
+          fetch-depth: 1
+      - run: git config user.name 'github-actions[bot]' && git config user.email 'github-actions[bot]@users.noreply.github.com'
+      - run: sudo apt-get update && sudo apt-get -yq install graphviz git fonts-ipafont fonts-ipaexfont
+      - run: >
+          wget -q -O /usr/local/bin/plantuml https://github.com/mikaelhg/puni2/releases/download/1.2022.7/plantuml-1.2022.7-glibc-x86_64
```

Comment:
I'm fine with using this version as I expect it is already used + tested somewhere else. But also please consider updating at some point the future as there are many [more recent releases](https://github.com/mikaelhg/puni2/releases).

Reply:
I assume that has been aligned with the version in Backstage. Not sure if that is still the case though.